docs/how-to/contribute.rst

```diff
@@ -0,0 +1,45 @@
+.. _contribute:
+
+============================
+How to contribute to Rubicon
+============================
+
+If you experience problems with Rubicon, `log them on GitHub`_. If you want
+to contribute code, please `fork the code`_ and `submit a pull request`_.
+
+.. _log them on Github: https://github.com/pybee/rubicon-objc/issues
+.. _fork the code: https://github.com/pybee/rubicon-objc
+.. _submit a pull request: https://github.com/pybee/rubicon-objc/pulls
```

Comment:
It feels weird to link to a page that's just the list of pulls, and doesn't really contain instructions on how to make a pull request.

Reply:
Agreed. We need to flesh this out a bit, and/or point to some generic "how to submit a PR" documentation. However, that doesn't exist (yet), and I want to get the basic document structure in place.